docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  parity-dev-node:
+    build: ./parity-dev-node
+    ports:
+      - "47622:8545"
+      - "47621:8180"
+  # get-dev-node:
```

Comment:
Minor typo: should be `# geth-dev-node:`.

Reply:
Fixed.